src/Compilers/CSharp/Test/Semantic/Semantics/LocalFunctionTests.cs

```diff
@@ -5545,6 +5614,97 @@ void M<T>([AllowNull] T t1, T t2)
                 Diagnostic(ErrorCode.WRN_NullReferenceArgument, "t1").WithArguments("t", "void local2(T t)").WithLocation(13, 16));
         }
 
+        [Fact]
+        public void LocalFunction_MaybeNullAttribute()
+        {
+            var source = @"
+#nullable enable
+
+using System.Diagnostics.CodeAnalysis;
+
+class C
+{
+    void M<TOuter>()
+    {
+        getDefault<string>().ToString(); // 1
+        getDefault<string?>().ToString(); // 2
+        getDefault<int>().ToString();
+        getDefault<int?>().ToString();
```

Comment:
Should this be `_ = getDefault<int?>.Value; // 3`

Reply:
Yeah it seems like that would be a more meaningful test.